t/40-check_qemu_oom.t

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env perl
```

Comment:
Wouldn't it make more sense to add this to `t/44-scripts.t` instead of having a separate file? But I guess it is ok, this test doesn't include much and has not expensive setup.

Reply:
t/44-scripts.t is intended to call every script in a loop in the same way, same as we have in openQA. So any specific tests should IMHO be in a separate test unit. In particular for os-autoinst we don't have any expensive database setup so the overhead of a separate file is really small.